core/src/i18n/en.yaml

```
@@ -146,6 +146,23 @@ applications:
     provisioning: The Application is being provisioned asynchronously.
 apps:
   title: Apps
+authorization-policies:
+  description: <0>Istio Authorization Policy</0> enables access control on workloads in the mesh.
```

```suggestion
  description: <0>Istio Authorization Policy</0> allows for workload access management in the mesh.
```
I've paraphrased a little bit to avoid having the same sentence as in the official Istio documentation.